package.json

```diff
@@ -8,7 +8,8 @@
   },
   "private": "true",
   "scripts": {
-    "test": "node ci/integration.js"
+    "test": "node ci/integration.js",
+    "test:server": "./ci/local.sh"
```

Comment:
Did you mean to add in `ci/clone_request.py` somewhere in here too?

Reply:
I had only been doing this interactively, I was going to get the functionality laid out before I moved forward with that.

Reply:
I added more documentation in baef651 to explain the purpose of this file. @stormpython Do you think it explains it thoroughly enough?

Reply:
Yes, the comments are sufficient.